### DISCUSSION TITLE
How to avoid highlighting operators?

### DISCUSSION THREAD

**Discussion by ssbarnea** [Textualize/rich#846] [Q&A] [answered]
I am printing a report line like below and one side effect of using rich is that the parentheses become highlighted, when in that case I would not want it to happen, as it does not make the test easier to read. On the same time I do want to keep the highlight of the numbers as this is useful:

```python
        console_stderr.print(
            f"Finished with {failures} failure(s), {warnings} warning(s) "
            f"on {len(result.files)} files."
        )

```
![](https://sbarnea.com/ss/Screen-Shot-2020-12-22-18-21-47.61.png)

Is that an easy way to avoid that? In fact I am even wondering if it would not be better to always use plurals and avoid the parentheses, with the risk of upsetting few grammar cops.

PS. I would want to avoid implementing a pluralize function.

**Comment by willmcgugan** [comment, marked as answer]
You could create a custom highlighter, based on [ReprHighlighter](https://github.com/willmcgugan/rich/blob/master/rich/highlighter.py#L79). i.e. copy it and remove the braces regex. See the second listing [here](https://rich.readthedocs.io/en/latest/highlighting.html) for how to use it.
